tests/fpm/php-7.2/php_version.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
```

Comment:
replaceable with:

```
set -e

php -v | grep "^PHP 7.2"
```

Reply:
I could, but then it would exit directly. I don't want that as there is a visual output below.
Plus, I want complete control on the output: 0 or 1 and nothing else.